src/model/queries/lastMessages.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
```

Comment:
why do we need this file??

Reply:
i was try something and forget remove these modules